GeofenceDisplay/test/verify/L.WatermarkControl.js

```diff
@@ -0,0 +1,17 @@
+L.Control.Watermark = L.Control.extend({
```

Comment:
why do we need to separately include this?

Reply:
Just there temporarily for my own testing and comparison reasons - it's a verified testable custom plugin. I'll take it out of the versioning as it's not directly relevant to the repo